beta-src/src/components/ui/WDCountdownPill.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from "react";
+import { useEffect, useState } from "react";
+import { Avatar, Chip } from "@mui/material";
+import BlackCountdownIcon from "../../assets/png/icn_phase_countdown_black.png";
+import RedCountdownIcon from "../../assets/png/icn_phase_countdown_red.png";
+import calculateAndSortRemainingTime from "../../utils/calculateAndSortRemainingTime";
+import { ParsedTime } from "../../interfaces/ParsedTime";
+import formatTime from "../../utils/formatTime";
+
+interface WDCountdownPillProps {
+  endTime: number;
+}
+
+const WDCountdownPill: React.FC<WDCountdownPillProps> = function ({ endTime }) {
+  const endTimeInMilliSeconds = endTime * 1000;
+
+  const [quarterTimeRemaining] = useState(
+    endTimeInMilliSeconds - (endTimeInMilliSeconds - +new Date()) / 4,
+  );
+
+  const [timeLeft, setTimeLeft] = useState<ParsedTime>(
+    calculateAndSortRemainingTime(endTime),
+  );
+
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      setTimeLeft(calculateAndSortRemainingTime(endTime));
+    }, 1000);
+
+    return () => clearTimeout(timer);
+  });
+
+  const chipDisplay: string = formatTime(timeLeft);
+  let chipColor = "black";
+  let image = BlackCountdownIcon;
+
+  if (+new Date() > quarterTimeRemaining) {
+    chipColor = "red";
+    image = RedCountdownIcon;
+  }
+
+  return (
+    <Chip
+      avatar={
+        <Avatar>
+          <img src={image} alt="" />
```

Comment:
As mentioned in a previous comment this should be a custom SVG component that changes color based on props. You should be able to find some examples of this in the codebase of how it can be done.

Reply:
Im not sure if you are referring to the img, but Elliot said to use the png because he can not make that an SVG